cypress/integration/deckBuilder/dryRun.js

```diff
@@ -125,4 +127,26 @@ describe('Deck Builder — Dry-run', () => {
       }
     )
   })
+
+  it('should not be possible to play Unhealthy Hysteria before turn 2', () => {
+    const UNHEALTHY_HYSTERIA_DECK =
+      'NU4xLDVOMiw1TjMsNU4yMyw1TjQsNU41LDVONiw1TjYyLDVONjMsNU42Nyw1TjY2LDVONw'
+    const HAND = ['N1', 'N2', 'N3', 'N63']
+
+    cy.visit(`/deck/${UNHEALTHY_HYSTERIA_DECK}/dry-run?mode=MANUAL`)
+
+      .drDrawHand(HAND)
+
+      .drSelect('N63')
+      .get(s.DR_PLAY_BTN)
+      .should('be.disabled')
+
+      .drEndTurn()
+
+      .drSelect('N63')
+      .get(s.DR_PLAY_BTN)
+      .should('not.be.disabled')
```

Comment:
So this is technically not necessary since `.drPlay('N63')` will fail if the play button is disabled.